src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,18 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
```

Comment:
this Component is redundant since you depend on a prop what to render. So it's better to move condition what to render to PostInfo directly